src/components/search/Search.jsx

```diff
@@ -22,7 +32,7 @@ function Search() {
                 ></path>
               </svg>
             </div>
-            <input type="text" class="input input-style input-search" />
+            <input type="text" class="input input-style input-search" value={filterState.search} onChange={changeHandler}/>
```

Comment:
When we type in search the first letter does not print, I think value should be search to solve this.

Reply:
fixed